examples/basic/src/actions/index.js

```diff
@@ -3,11 +3,10 @@ import * as Rx from 'rxjs';
 export const FETCH_USER_PENDING = 'FETCH_USER_PENDING';
 export const FETCH_USER_FULFILLED = 'FETCH_USER_FULFILLED';
 export const FETCH_USER_ABORTED = 'FETCH_USER_ABORTED';
+const BASE_URL = 'http://localhost:3000/items/';
 
 export const fetchUser = () => (
-  (actions, store) => Rx.Observable.of({ id: 1, name: 'Bilbo Baggins', timestamp: new Date() })
-    // Delaying to emulate an async request, like Rx.Observable.ajax('/api/path')
-    .delay(1000)
+  (actions, store) => Rx.Observable.ajax(BASE_URL)
```

Comment:
ahh wait, the timestamp is not fixed correctly!  Will fix shortly.

On Tue, Jun 7, 2016 at 1:22 PM, Jay Phelps <notifications@github.com> wrote:

> In examples/basic/src/actions/index.js
> <https://github.com/redux-observable/redux-observable/pull/43#discussion_r66114198>
> :
>
> >
> >  export const fetchUser = () => (
> > -  (actions, store) => Rx.Observable.of({ id: 1, name: 'Bilbo Baggins', timestamp: new Date() })
> > -    // Delaying to emulate an async request, like Rx.Observable.ajax('/api/path')
> > -    .delay(1000)
> > +  (actions, store) => Rx.Observable.ajax(BASE_URL)
>
> Cool. Will test locally this afternoon to confirm it works as expected and
> then merge. 👍
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/redux-observable/redux-observable/pull/43/files/8f55d936d43882bd2008e2f90b71fd09a602b26e#r66114198>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAt8T_z7H4e171BBse-s8h8d-wNWNh5yks5qJajOgaJpZM4IwCId>
> .
>